defaults/main.yml

```diff
@@ -40,6 +40,8 @@ telegraf_influxdb_password:
 telegraf_influxdb_user_agent:
 telegraf_influxdb_udp_payload:
 
+telegraf_prometheus_metrics_filtered: []
```

Comment:
Should we simply say `telegraf_metrics_filtered`?

Reply:
and put `namedrop` in `output.influxdb` as well?

Reply:
I think so, yes.

Reply:
👍